runtime/run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo "suid"
+./cli suid /tmp/testing.simg /bin/ls /
```

Comment:
What work is left to be done to run SIF images here? I assume we need to use SIF binary to expose/mount the underlying FS?

Reply:
Yes, Yannick could confirm, but I believe we need sif_load in core/lib/sif/sif.c to read SIF image and get data object descriptors to identify partitions and their offsets to mount correct ones